src/gui/Src/Gui/MainWindow.cpp

```diff
@@ -294,7 +294,7 @@ MainWindow::MainWindow(QWidget* parent)
     connect(ui->actionLabels, SIGNAL(triggered()), this, SLOT(displayLabels()));
     connect(ui->actionBookmarks, SIGNAL(triggered()), this, SLOT(displayBookmarks()));
     connect(ui->actionFunctions, SIGNAL(triggered()), this, SLOT(displayFunctions()));
-    connect(ui->actionCheckUpdates, SIGNAL(triggered()), this, SLOT(checkUpdates()));
+    //connect(ui->actionCheckUpdates, SIGNAL(triggered()), this, SLOT(checkUpdates()));

```

Comment:
The connect can just be removed since it's no longer relevant once this is merged.